ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1683,12 +1716,36 @@ def test_frontend_method(
             **kwargs_method,
         )
 
+        # frontend ret
+        # ret = get_frontend_ret(
+        #     backend_to_test,
+        #     ins.__getattribute__(frontend_method_data.method_name),
+        #     *args_method_ivy,
+        #     # test_compile=test_flags.test_compile,
+        #     # frontend_array_function=(
+        #     #     create_frontend_array if test_flags.test_compile else None
+        #     # ),
+        #     as_ivy_arrays=(not method_flags.generate_frontend_arrays),
+        #     precision_mode=method_flags.precision_mode,
+        #     **kwargs_method_ivy,
+        # )
+
         # ToDo: uncomment once test_frontend_method has been updated to test for
         #  frontend array arguments like test_frontend_function where
         #  test_flags.generate_frontend_arrays is being used
-        # assert ivy_backend.nested_map(
-        #     ret, lambda x: _is_frontend_array(x) if ivy_backend.is_array(x) else True
-        # ), "Frontend method returned non-frontend arrays: {}".format(ret)
+        assert ivy_backend.nested_map(
+            ret, lambda x: _is_frontend_array(x) if ivy_backend.is_array(x) else True
+        ), "Frontend method returned non-frontend arrays: {}".format(ret)
+
+        # create NumPy args
+        if method_flags.generate_frontend_arrays:
+            ret_np_flat = flatten_frontend_to_np(
+                ret=ret,
+                frontend_array_fn=create_frontend_array,
+                backend=backend_to_test,
+            )
+        else:
+            ret_np_flat = flatten_and_to_np(ret=ret, backend=backend_to_test)
```

Comment:
Dont think this is necessary, `ret_np_flat` is being computed above.
and `ret` will always be an ivy array, since `get_ret_and_flattened_np_array` converts it to an ivy array.